src/core/lib/iomgr/closure.h

```diff
@@ -42,7 +42,9 @@ typedef struct grpc_closure_list {
  *
  * \param arg Arbitrary input.
  * \param error GRPC_ERROR_NONE if no error occurred, otherwise some grpc_error
- *              describing what went wrong */
+ *              describing what went wrong.
+ *              Error contract: it is not the cb's job to unref this error;
```

Comment:
Good point, but I feel that having it spelled out in the code is also useful.

Note that the other cases in this PR are not explicitly covered by the document: the document says that ownership transfers when an error is passed to a function (which sounds like an argument), but those other cases have a struct as their argument, one of whose members is the error.